sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_properties_async.py

```diff
@@ -18,6 +19,40 @@
     assert properties['eventhub_name'] == live_eventhub['event_hub'] and properties['partition_ids'] == ['0', '1']
     await client.close()
 
+@pytest.mark.liveTest
+@pytest.mark.asyncio
+async def test_get_properties_with_auth_error_async(live_eventhub):
+    client = EventHubConsumerClient(live_eventhub['hostname'], live_eventhub['event_hub'], '$default',
+        EventHubSharedKeyCredential(live_eventhub['key_name'], "AaBbCcDdEeFf=")
+    )
+    with pytest.raises(AuthenticationError) as e:
+        await client.get_eventhub_properties()
+    await client.close()
+
+    client = EventHubConsumerClient(live_eventhub['hostname'], live_eventhub['event_hub'], '$default',
+        EventHubSharedKeyCredential("invalid", live_eventhub['access_key'])
+    )
+    with pytest.raises(AuthenticationError) as e:
+        await client.get_eventhub_properties()
+    await client.close()
```

Comment:
Better use try ... finally or "with" statement to close this client in case the test assertion fails to make sure client is closed after test, either success or fail.
I see some old test code has the same problem. I'm going to change it too.